### ISSUE TITLE
WCAG success criteria hyperlink is missing for the primary WCAG Success Criteria

### ISSUE THREAD

**Issue by PriyankaUofI** [opena11y/ainspector-for-firefox#55]
Test URL : https://cerebro.test.iamu.techservices.illinois.edu/#userinformation
The first success criteria of any issue is not hyperlinked. It was a hyperlink in previous versions. Is this removed on purpose?

![image](https://github.com/opena11y/ainspector-for-firefox/assets/84582289/1887723a-0c66-4bca-9bf4-e09be74a5ebe)

![image](https://github.com/opena11y/ainspector-for-firefox/assets/84582289/0d39905f-b789-4905-a346-884ca52585b7)

**Comment by jongund** [comment]
@PriyankaUofI 
Thank you for reporting, will get this fixed.

**Comment by jongund** [comment]
@PriyankaUofI 
This was fixed in [AInspector 2.99.1](https://addons.mozilla.org/en-US/firefox/addon/ainspector-wcag/).
